server/backend/quiz/models.py

```
@@ -0,0 +1,51 @@
+    text = models.TextField()
+    created_at = models.DateTimeField(auto_now_add=True)
+
+    def __str__(self):
+        return self.text[:50]
+
+class Option(models.Model):
+    question = models.ForeignKey(Question, related_name='options', on_delete=models.CASCADE)
+    text = models.CharField(max_length=255)
+    is_correct = models.BooleanField(default=False)
+
+    def __str__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__str__`

Add return type annotation: `str`

(ANN204)